### ISSUE TITLE
CompileLibraryResourcesTask warning even when experimental properties are set

### ISSUE THREAD

**Issue by eygraber** [gradle/android-cache-fix-gradle-plugin#234]
I use version 2.4.6 of this plugin, and enable the experimental properties that AGP added in `gradle.properties` for both my main project and an included build:

```
android.experimental.cacheCompileLibResources=true
android.experimental.enableSourceSetPathsMap=true
```

It seems like when the included build is built completely from scratch (not incremental or in the build cache), this warning gets printed anyways.

```
> Task :my-included-build:generatePrecompiledScriptPluginAccessors
WARNING: Android plugin 7.2.0-beta01 has experimental support for using relative path sensitivity with CompileLibraryResourcesTask inputs which will provide more build cache hits and improve build speed.  Set 'android.experimental.cacheCompileLibResources=true' and 'android.experimental.enableSourceSetPathsMap=true' in gradle.properties to enable this support.
Android plugin 7.2.0-beta01 has experimental support for using relative path sensitivity with CompileLibraryResourcesTask inputs which will provide more build cache hits and improve build speed.  Set 'android.experimental.cacheCompileLibResources=true' and 'android.experimental.enableSourceSetPathsMap=true' in gradle.properties to enable this support.
```

**Comment by runningcode** [comment]
Thanks for reporting this. Do you know if you are experiencing this warning when the configuration cache is enabled?

Are you able to reproduce this in a smaller sample project?

**Comment by eygraber** [comment]
The configuration cache is enabled. 

I haven't tried to repro in a smaller project yet.

**Comment by ghale** [comment]
@eygraber I have tried to reproduce this with an included build and configuration cache and I have been unsuccessful.  The only scenario I was able to see something similar is when the properties are missing from an included build.  However, in such a case, the warning appears on every build, not intermittently.  

I suspect there is some detail that I'm missing.  Is there anything else about your build that might be interesting?  Again, if you can provide a reproducer, that would go a long way towards finding/fixing the error.

**Comment by eygraber** [comment]
I can repro it in this attached project by running `gradle :my-gradle:generatePrecompiledScriptPluginAccessors`

[androidcachefix234.zip](https://github.com/gradle/android-cache-fix-gradle-plugin/files/8415513/androidcachefix234.zip)

**Comment by ghale** [comment]
Thanks @eygraber.  I see the problem now.  Might be a little tricky to fix, but I at least understand what's going on now.
